docs/docs/contributing/translations.mdx

```diff
@@ -58,7 +58,7 @@ for your target language already exists in the `superset/translations` directory
 ls superset/translations | grep -E "^[a-z]{2}\/"
 ```
 
-If your language already has a pre-existing translation, skip to the next section
+If your language already has a preexisting translation, skip to the next section
```

Comment:
I'm happy either way, but this one is a funny case! It depends on whether we're using the AP or Chicago style manuals ;)